content/articles/building-a-video-cataloging-app-with-react-firebase-and-styled-components/index.md

```diff
@@ -0,0 +1,543 @@
+    id: getUUID(),
+    title: "Maintenance Services",
+    description:
+      "Introduction to Maintenance Services, concept of Maintenance Services, classifications, principles and related theories",
+    genre: "Educational",
+    duration: "3 hrs",
+    slug: "maintenance-services",
+  });
+
+  // Action
+  firebase.firestore().collection("movies").add({
+    id: getUUID(),
+    title: "The Amazing Spiderman",
+    description:
+      "The Amazing Spiderman franchise has been widely popular among action movies enthusiasts. It is produced by Marvel studios",
+    genre: "action",
+    duration: "5 hrs",
+    slug: "the-amazing-spiderman",
+  });
+
+  /* series
+    ============================================ */
+  // Drama
+  firebase.firestore().collection("series").add({
+    id: getUUID(),
+    title: "Squid Game",
+    description:
+      "Squid game is a popular Korean 2021 series. It is now the most-watched series on Netflix with billions of streamed hours",
+    genre: "drama",
+    duration: "6 hrs",
+    slug: "squid-game",
+  });
+  firebase.firestore().collection("series").add({
+    id: getUUID(),
+    title: "Harry Potter",
+    description:
+      "Harry Potter is an incredible series about mysteries and magic in England from the book Harry Potter.",
+    genre: "drama",
+    duration: "8 hrs",
+    slug: "harry-potter",
+  });
+
+  // Suspense
+  firebase.firestore().collection("series").add({
+    id: getUUID(),
+    title: "Knives out",
+    description:
+      "Knives Out is an interesting suspense movie about a murder incident involving a famous multi-millionaire author in his home estate.",
+    genre: "suspense",
+    duration: "1 hrs 30 mins",
+    slug: "knives-out",
+  });
+
+  // Children
+  firebase.firestore().collection("series").add({
+    id: getUUID(),
+    title: "Ben 10",
+    description:
+      "Ben 10 is an interesting kids movie about a little boy who found a watch that gave him super-human abilities to transform into many creatures. Ben alongside his grandpa and sister decided to use their new abilities to fight crime.",
+    genre: "children",
+    duration: "2 hrs",
+    slug: "ben-10",
+  });
+
+  // Romance
+  firebase.firestore().collection("series").add({
+    id: getUUID(),
+    title: "Romeo and Juliet",
+    description:
+      "Romeo and Juliet is a classic romance movie from the legendary novel Romeo and Juliet.",
+    genre: "Romance",
+    duration: "2 hrs",
+    slug: "romeo-and-juliet",
+  });
+}
+```
+
+The snippet above will simply create a firebase catalog containing the above-listed categories and video information.
+To upload the content to firebase, an internet connection is required. Once you are connected to a Wi-Fi or Modem, save the `catalog.js` file and refresh the code editor, which should automatically upload everything to the Firebase project you created earlier. so open up the firebase tab in your browser and view the catalog.

```

Comment:
Saving a file does not trigger any action other than the file being saved onto your local system.

In this section, you have a function, `seedDatabase` that would need to be run before anything can happen in regards to the creation of the firebase catalog. But in your tutorial, the function `seedDatabase` is only defined. Never run. 

This means that even after a reader saves the catalog.js file and "refreshes the code editor" (I'm not sure what you mean by that), they would not find the catalog created on firebase.